src/Symfony/Component/Dotenv/CHANGELOG.md

```diff
@@ -4,4 +4,5 @@ CHANGELOG
 3.3.0
 -----
 
+ * [BC BREAK] Switched behavior of the `Dotenv` class to override previously loaded envs
```

Comment:
[BC BREAK] Since v3.3.7, the latest `Dotenv` files override the previous ones. Real env vars are not affected and are not overridden.

Reply:
@nicolas-grekas, fixed! Thanks!